src/isar/apis/models/start_mission_definition.py

```diff
@@ -52,7 +50,7 @@ class StartMissionInspectionDefinition(BaseModel):
 class StartMissionTaskDefinition(BaseModel):
     type: TaskType = Field(default=TaskType.Inspection)
     pose: InputPose
-    inspections: List[StartMissionInspectionDefinition]
+    inspection: StartMissionInspectionDefinition
```

Comment:
This part is about changing the mission structure and should be hidden within the remove step commit